src/arrays/referencing_array_and_its_elements/hard/q4/PotentialGoodStudents.java

```diff
@@ -0,0 +1,34 @@
+package arrays.referencing_array_and_its_elements.hard.q4;
+
+/*
+Write a Java program that takes an array of 10 doubles between 0 and 100, indicating course grades for ten students.
```

Comment:
Change question text to:
`Write a Java program that takes an array of 10 doubles between 0 and 100, indicating course grades for ten students. Then, traverse through the array and print out the index of students on the same line separated by spaces (Eg: "3 4 6 " ) who got above 88% in the course, indicating they are potential candidates for teaching assistants next year. To do this, create a method called PGS that takes the double array of grades and prints out the indices of the candidate students. Your method should be general enough to work with any length of array.`